lib/plugin/screenshotOnFail.js

```diff
@@ -131,4 +131,16 @@ module.exports = function (config) {
       }
     }, true);
   });
+
+  function _getUUID(test) {
```

Comment:
Do we need jsdoc and typings for this function?

You can make jsdoc and mark it as `@private`

Reply:
ah no need, it's just being used in this.